### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by dermyhancock**
## Pull Request #1
This PR updates the caption, out of date.
This PR closed/resolves/fixes #2 

## Done
- created a branch
- made a commit

## To do

- [ ] merge the branch
- [ ] delete the ## My Pull Request

### CODE REVIEW

File: _slides/15-dermyhancock.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+My first caption
```
Comment (howardpurchase) [inline _slides/15-dermyhancock.md:6]: Nice caption!

File: _slides/15-dermyhancock.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-![cherryontop-o-cat](https://octodex.github.com/images/cherryontop-o-cat.png)
```
Comment (howardpurchase) [inline _slides/15-dermyhancock.md:8]: Nothing to suggest

**Review by howardpurchase — APPROVED** [review]
Great job Derrick!!

File: _slides/15-dermyhancock.md (hunk 1 above)
Comment (matt877) [inline _slides/15-dermyhancock.md:6]: What an epic caption

Comment (matt877) [inline _slides/15-dermyhancock.md:6]:
```suggestion
My first caption - I suggest nothing,
```

**Review by matt877 — APPROVED** [review]
Dandy

File: _slides/15-dermyhancock.md (hunk 1 above)
Comment (GaryWKavanagh) [inline _slides/15-dermyhancock.md:6]: This is good.

**Review by GaryWKavanagh — APPROVED** [review]
Approved.

File: _slides/15-dermyhancock.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (Silver709) [inline _slides/15-dermyhancock.md:6]: Everything looks good

File: _slides/15-dermyhancock.md (hunk 1 above)
Comment (Silver709) [inline _slides/15-dermyhancock.md:6]:
```suggestion
Random suggestion here
```

**Review by Silver709 — APPROVED** [review]
All is well - APPROVED

**Pull request merged**